tests/attocube.py

```diff
@@ -40,12 +41,20 @@ def setUp(self):
         self.ca.assert_that_pv_exists(MOTOR_RBV)
 
     def test_WHEN_moved_to_position_THEN_position_reached(self):
-        position_setpoint = 10
+        position_setpoint = 20
         self.ca.set_pv_value(MOTOR_PV, position_setpoint)
         self.ca.assert_that_pv_value_is_increasing(MOTOR_RBV, 1)
-        self.ca.assert_that_pv_is_number(MOTOR_RBV, position_setpoint, timeout=10)
+        self.ca.assert_that_pv_is_number(MOTOR_RBV, position_setpoint, timeout=20)
 
-    @unittest.skipIf(True, 'This should work but comms alarms seem to be somewhat broken see #4618')
     def test_GIVEN_device_not_connected_THEN_pv_in_alarm(self):
         self._lewis.backdoor_set_on_device('connected', False)
-        self.ca.assert_that_pv_alarm_is(MOTOR_PV, ChannelAccess.Alarms.INVALID, timeout=60)
+        self.ca.assert_that_pv_alarm_is(MOTOR_PV, ChannelAccess.Alarms.INVALID, timeout=30)
+        self._lewis.backdoor_set_on_device('connected', True)
+
+    def test_GIVEN_device_not_connected_WHEN_motor_val_set_THEN_pv_returns_to_alarm(self):
+        self._lewis.backdoor_set_on_device('connected', False)
+        time.sleep(10)
```

Comment:
Best to avoid arbitrary sleeps, instead wait for what you want to happen with a timeout. I assume we're waiting for an alarm to occur here so do `self.ca.assert_that_pv_alarm_is(MOTOR_PV, ChannelAccess.Alarms.INVALID, timeout=10)`. This will mean that if it takes less than 10 seconds it will not waste time.